centralsettlement/values.yaml

```diff
@@ -54,7 +54,12 @@ config:
     {
       "PORT": $service_api_port,
       "HOSTNAME": "http://central-settlements.local",
-      "DATABASE_URI" : "$dbdriver://$dbuser:$dbpassword@$dbhost:$dbport/$dbname"
+      "DATABASE_URI" : "$dbdriver://$dbuser:$dbpassword@$dbhost:$dbport/$dbname",
+      "TRANSFER_VALIDITY_SECONDS": "432000",
+      "HUB_PARTICIPANT": {
```

Comment:
Any reason this is here? Should this not be determined by the data stored in the DB?